blob/blobtest/testing.go

```diff
@@ -1,38 +1,47 @@
 package blobtest
 
 import (
-	tmrand "github.com/tendermint/tendermint/libs/rand"
+	"encoding/binary"
+	"fmt"
 
-	"github.com/celestiaorg/celestia-app/v2/pkg/appconsts"
-	"github.com/celestiaorg/celestia-app/v2/test/util/testfactory"
-	"github.com/celestiaorg/go-square/blob"
-	"github.com/celestiaorg/go-square/shares"
+	tmrand "github.com/tendermint/tendermint/libs/rand"
 
-	"github.com/celestiaorg/celestia-node/share"
+	"github.com/celestiaorg/celestia-app/v3/test/util/testfactory"
+	gosquare "github.com/celestiaorg/go-square/v2/share"
 )
 
 // GenerateV0Blobs is a test utility producing v0 share formatted blobs with the
 // requested size and random namespaces.
-func GenerateV0Blobs(sizes []int, sameNamespace bool) ([]*blob.Blob, error) {
-	blobs := make([]*blob.Blob, 0, len(sizes))
-
+func GenerateV0Blobs(sizes []int, sameNamespace bool) ([]*gosquare.Blob, error) {
+	blobs := make([]*gosquare.Blob, 0, len(sizes))
 	for _, size := range sizes {
-		size := rawBlobSize(appconsts.FirstSparseShareContentSize * size)
+		size := RawBlobSize(gosquare.FirstSparseShareContentSize * size)
 		appBlob := testfactory.GenerateRandomBlob(size)
 		if !sameNamespace {
-			namespace, err := share.NewBlobNamespaceV0(tmrand.Bytes(7))
+			namespace, err := gosquare.NewV0Namespace(tmrand.Bytes(7))
+			if err != nil {
+				return nil, err
+			}
+			if namespace.IsReserved() {
+				return nil, fmt.Errorf("reserved namespace")
+			}
```

Comment:
This is gonna make the test flaky; if the rand bytes turn out to be reserved, it will error. Loop until it's valid instead?